docs/tutorial/quantum_annealing.ipynb

```diff
@@ -0,0 +1,518 @@
+    "$$\n",
+    "H(s) = s H_0 + (1 - s) H_1\n",
+    "$$\n",
+    "\n",
+    "where $ H_0 $ is the Hamiltonian for the Max-Cut problem we want to solve and often formulated in terms of an Ising model, and $ H_1 $ is the Hamiltonian for the transverse magnetic field.For further details on how to construct the Hamiltonian for the max-cut problem, please refer to [QAOA for Max-Cut](https://jij-inc.github.io/Qamomile/tutorial/maxcut.html).\n",
+    "\n",
+    "$$\n",
+    "H_0 = \\frac{1}{2} \\sum_{(i,j) \\in E} w_{i,j}(s_i s_j - 1)  , \\quad H_1 = - \\sum_{i=1}^N \\sigma_i^x\n",
+    "$$\n",
+    "\n",
+    "\n",
+    "As $t$ increases and $1-t$ decreases, the system gradually evolves from the ground state of $H_1$ to the ground state of  $H_0$, which corresponds to the solution to the optimization problem. \n",
+    "\n",
+    "Quantum annealing relies on quantum tunneling to allow the system to escape local minima by passing through energy barriers, rather than needing to climb over them as in classical annealing.\n",
+    "\n",
+    "First, let’s import the Qamomile library and other libraries used in this example.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "be3ddbdf-1794-4c7e-96eb-5eb0a3994bf0",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$$\\begin{array}{cccc}\\text{Problem:} & \\text{Maxcut} & & \\\\& & \\min \\quad \\displaystyle 0.5 \\cdot \\sum_{e \\in E} w_{e_{0}, e_{1}} \\cdot \\left(s_{e[0]} \\cdot s_{e[1]} + 1\\right) & \\\\\\text{{where}} & & & \\\\& x & 1\\text{-dim binary variable}\\\\\\end{array}$$"
+      ],
+      "text/plain": [
+       "<jijmodeling.Problem at 0x16a27f0f0>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import jijmodeling as jm\n",
+    "import numpy as np\n",
+    "import qamomile.core.operator as qm_o\n",
+    "import matplotlib.pyplot as plt\n",
+    "import qamomile.qutip as qm_qt\n",
+    "\n",
+    "# N = 4\n",
+    "# J = {}\n",
+    "# h = {}\n",
+    "\n",
+    "# for i in range(N-1):\n",
+    "#     for j in range(i+1, N):\n",
+    "#         J[(i, j)] = random.gauss(0, 1)\n",
+    "\n",
+    "# for i in range(N):\n",
+    "#     h[i] = random.gauss(0, 1)\n",
+    "\n",
+    "def Maxcut_problem() -> jm.Problem:\n",
+    "    V = jm.Placeholder(\"V\")\n",
+    "    E = jm.Placeholder(\"E\", ndim=2)\n",
+    "    w = jm.Placeholder(\"w\", ndim=2)\n",
+    "    x = jm.BinaryVar(\"x\", shape=(V,))\n",
+    "    e = jm.Element(\"e\", belong_to=E)\n",
+    "    i = jm.Element(\"i\", belong_to=V)\n",
+    "    j = jm.Element(\"j\", belong_to=V)\n",
+    "\n",
+    "    problem = jm.Problem(\"Maxcut\")\n",
+    "    si = 2 * x[e[0]] - 1\n",
+    "    sj = 2 * x[e[1]] - 1\n",
+    "    si.set_latex(\"s_{e[0]}\")\n",
+    "    sj.set_latex(\"s_{e[1]}\")\n",
+    "    obj = 1 / 2 * jm.sum(e, w[e[0], e[1]] * (1 + si * sj))\n",
```

Comment:
This is incorrect.
The sign in the sum is different

Reply:
Do you mean it should be ` obj = 1 / 2 * jm.sum(e, w[e[0], e[1]] * (1 - si * sj)` ? because I change from `max` to `min`

Reply:
I think the original formulation of Maxcut problem is $\max \frac{1}{2}\sum_{(i,j)\in E}(1 - s_is_j)$ because we want to count up the edges which connect different group of nodes.
The convertion to minimization problem makes $$\min -\frac{1}{2}\sum_{(i,j)\in E}(1 - s_is_j)$$, not $$\min \frac{1}{2}\sum_{(i,j)\in E}(1 + s_is_j)$$

Reply:
got it!